third_party/openmetadata/prepare_OM_for_fybrik.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	logging "fybrik.io/fybrik/pkg/logging"
+	openapi_connector_core "fybrik.io/openmetadata-connector/pkg/openmetadata-connector-core"
+)
+
+func main() {
+	conf := map[interface{}]interface{}{
+		"openmetadata_endpoint": "http://localhost:8585/api",
```

Comment:
can it be an argument or env parameter

Reply:
The preparation code gets the endpoint, username, and password from an environment variable.